experimental/segmenter/src/line.rs

```diff
@@ -109,12 +110,15 @@ impl Default for LineBreakOptions {
 pub struct LineBreakSegmenter {
     options: LineBreakOptions,
     payload: DataPayload<LineBreakDataV1Marker>,
+    trie_payload: DataPayload<UCharDictionaryBreakDataV1Marker>,
 }
 
 impl LineBreakSegmenter {
     pub fn try_new<D>(provider: &D) -> Result<Self, DataError>
     where
-        D: ResourceProvider<LineBreakDataV1Marker> + ?Sized,
+        D: ResourceProvider<LineBreakDataV1Marker>
+            + ResourceProvider<UCharDictionaryBreakDataV1Marker>
+            + ?Sized,
```

Comment:
Note: We should make a plan for how to make segmentation data configurable. Not in this PR.

Reply:
Although LSTM segmenter isn't target for 1.0, we have to think LSTM data too for line and word.